website/docs/community/contribute.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Become a contributor
```

Comment:
I (and the Docs team) deeply dislike having pages which are only accessible by clicking the header, especially when Additional resources doesn't behave the same way and just toggles expanded/contracted state
![image](https://user-images.githubusercontent.com/7335046/194965738-a400816b-9e87-45ea-bcbb-749b3ee215f0.png)

Could this instead switch to be a fifth page called `Become a Contributor`?

Reply:
strong disagree! but i'm not that invested in it, so sure